.github/workflows/pull_request.yml

```diff
@@ -73,8 +73,8 @@ jobs:
         run: |
           pip install mypy
           mypy --namespace-packages $SOURCE_DIRS | tee mypy.log
-          if [ -n "$( tail -n 1 mypy.log | grep -e '^Succes' )" ]; then RESULT="passed"; else RESULT="failed"; fi
-          if [ "${{ matrix.python-version }}" = "3.10" ]; then anybadge -o -l mypy -v $RESULT -f $BADGES_DIR/mypy.svg failed=red passed=green; fi
+          if [ -n "$( tail -n 1 mypy.log | grep -e '^Succes' )" ]; then RESULT="passing"; else RESULT="failing"; fi
+          if [ "${{ matrix.python-version }}" = "3.10" ]; then anybadge -o -l mypy -v $RESULT -f $BADGES_DIR/mypy.svg failing=red passing=green; fi
```

Comment:
The mypy tests either "passed" or "failed". They are not "passing" or "failing" at any moment. ;) Please change this back.

Reply:
the common convention it seems is to use "passing" and "failing" instead of the past tense

Reply:
As discussed within the team, we now choose to go our own way with "pass" and "fail"